<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QComboBox" name="FacultiescomboBox"/>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="DepartmentscomboBox"/>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="Add_Student_pushButton">
      <property name="text">
       <string>Додати студента</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2" colspan="4">
     <widget class="QTableWidget" name="tableWidget">
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>588</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Адміністрування</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="nononoact"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Дадати студента</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Адміністрування</string>
   </property>
  </action>
  <action name="nononoact">
   <property name="text">
    <string>Ви не володієте достатнім рівнем доступу</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
